### PR TITLE
[DOCS-6293] Outlook and teams connector

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

**Review by nazarethjim — APPROVED** [review]

File: process-automation/latest/model/connectors/teams.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
+title: Microsoft Teams connector
+---
+
+The Microsoft Teams connector uses the [Microsoft Graph](https://slack.com){:target="_blank"} API to intergrate with Microsfot Teams.
+
+The Teams connector is displayed on the process diagram with the Teams logo.
+
+> **Important**: The Teams connector and user both require a Microsoft Teams client. The Teams connector requires a Microsoft Teams account and is separate to the Alfresco hosted environment and should be created and managed by an administrator.
```
Comment (anxumalo) [inline process-automation/latest/model/connectors/teams.md:9]: I think you can split the last sentence so it's easier to read

**Review by anxumalo — CHANGES_REQUESTED** [review]
Looks good - built docs, ran Screaming Frog and checked for broken links/redirects. Other comments added in-line.

Additional fixes:
* http://localhost:4000/process-automation/latest/develop/#custom-form-fields - broken link (form field customizations)
* http://localhost:4000/process-automation/latest/model/connectors/aws/#bpmn-tasks-configuration - broken link (textAnalysisProcess.bpmn20.xml). It might be better to remove the hyperlink as it points to a private repo.
* http://localhost:4000/process-automation/latest/model/connectors/docusign/#configuration
** update redirect (Configure an application for JWT authentication)
** update redirect (Grant consent to the application)
* The Support site redirect gives an error: https://myalfresco.force.com/support/ - let's switch the URL to https://support.alfresco.com/

File: process-automation/latest/model/connectors/teams.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
+---
+title: Microsoft Teams connector
+---
+
+The Microsoft Teams connector uses the [Microsoft Graph](https://slack.com){:target="_blank"} API to intergrate with Microsfot Teams.
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply process-automation/latest/model/connectors/teams.md]: Corrected

File: process-automation/latest/model/connectors/outlook.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
+---
+title: Microsoft Outlook connector
+---
+
+The Outlook connector uses the [Microsoft Graph](https://slack.com){:target="_blank"} API to integrate with Microsoft Outlook.
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply process-automation/latest/model/connectors/outlook.md]: Corrected

File: process-automation/latest/model/connectors/teams.md (hunk 1 above)
Comment (oliveralfresco2018) [reply process-automation/latest/model/connectors/teams.md]: Adjusted

**Comment by oliveralfresco2018** [comment]
> * http://localhost:4000/process-automation/latest/develop/#custom-form-fields - broken link (form field customizations) Added a new link
> * http://localhost:4000/process-automation/latest/model/connectors/aws/#bpmn-tasks-configuration - broken link (textAnalysisProcess.bpmn20.xml). It might be better to remove the hyperlink as it points to a private repo. Removed
> * http://localhost:4000/process-automation/latest/model/connectors/docusign/#configuration
>   ** update redirect (Configure an application for JWT authentication) Done
>   ** update redirect (Grant consent to the application) Done
> * The Support site redirect gives an error: https://myalfresco.force.com/support/ - let's switch the URL to https://support.alfresco.com/ Done, search couldn't find for some reason!

**Review by anxumalo — COMMENTED** [review]
Please see inline comments - minor questions & tweaks

File: process-automation/latest/develop/index.md [hunk 4]
```diff
@@ -209,7 +209,7 @@ Once the custom interface has been fully developed and tested it can be deployed
 
 ## Custom form fields
 
-To include custom form fields within a form, the [form field customizations](https://www.alfresco.com/abn/adf/docs/user-guide/aae-extensions/){:target="_blank"} must be included in the [customization of Digital Workspace](#extend-the-digital-workspace) or the [development of a custom user interface](#develop-a-custom-user-interface).
+To include custom form fields within a form, the [form field customizations](https://github.com/Alfresco/alfresco-ng2-components/blob/develop/docs/user-guide/extensibility.md){:target="_blank"} must be included in the [customization of Digital Workspace](#extend-the-digital-workspace) or the [development of a custom user interface](#develop-a-custom-user-interface).
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply process-automation/latest/develop/index.md:212]: I was given it by Eugenio and i asked him that question too. 
hes ok with it :-)

File: process-automation/latest/model/connectors/aws.md [hunk 5]
```diff
@@ -343,8 +343,6 @@ If the asynchronous job finishes successfully a compressed output file (`output.
 
 ### BPMN Tasks Configuration
 
-The following [textAnalysisProcess.bpmn20.xml](https://github.com/Alfresco/alfresco-process-connector-services/blob/develop/alfresco-process-comprehend-connector-spring-boot-starter/textAnalysisProcess.bpmn20.xml) describes an example of how the text analysis connector is setup in AAE:
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply process-automation/latest/model/connectors/aws.md:346]: haha sorry done

**Review by anxumalo — APPROVED** [review]

**Pull request merged**